TailLight/device.h

```diff
@@ -1,13 +1,37 @@
 #pragma once
 
+#include "debug.h"
+
+#define MSINTELLIMOUSE_USBINTERFACE5_PREFIX L"\\??\\HID#VID_045E&PID_082A&MI_01&Col05"
```

Comment:
I would prefer to avoid hardcoding the HW ID here, since it will need to be manually kept in sync with the INX file. Would it be possible for you to instead retrieve it using `IoGetDevicePropertyData` or similar as described in https://learn.microsoft.com/en-us/windows-hardware/drivers/install/hardware-ids ?